<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select class:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Classes">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Nur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>II</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>III</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VII</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VIII</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XII</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Submit3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>287</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Submit3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
